connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectProtoSuite.scala

```diff
@@ -30,181 +36,140 @@ import org.apache.spark.sql.catalyst.plans.logical.LocalRelation
-        import org.apache.spark.sql.connect.dsl.plans._
-        transform(connectTestRelation.join(connectTestRelation2, t))
-      }
-      val sparkPlan3 = sparkTestRelation.join(sparkTestRelation2, y)
-      comparePlans(connectPlan3.analyze, sparkPlan3.analyze, false)
-    }
 
-    val connectPlan4 = {
-      import org.apache.spark.sql.connect.dsl.plans._
-      transform(
-        connectTestRelation.join(connectTestRelation2, JoinType.JOIN_TYPE_INNER, Seq("name")))
+      val connectPlan3 = connectTestRelation.join(connectTestRelation2, t, Seq("id"))
+      val sparkPlan3 = sparkTestRelation.join(sparkTestRelation2, Seq("id"), y.toString)
+      comparePlans(connectPlan3, sparkPlan3)
     }
-    val sparkPlan4 = sparkTestRelation.join(sparkTestRelation2, UsingJoin(Inner, Seq("name")))
-    comparePlans(connectPlan4.analyze, sparkPlan4.analyze, false)
+
+    val connectPlan4 =
+      connectTestRelation.join(connectTestRelation2, JoinType.JOIN_TYPE_INNER, Seq("name"))
+    val sparkPlan4 = sparkTestRelation.join(sparkTestRelation2, Seq("name"), Inner.toString)
+    comparePlans(connectPlan4, sparkPlan4)
   }
 
   test("Test sample") {
-    val connectPlan = {
-      import org.apache.spark.sql.connect.dsl.plans._
-      transform(connectTestRelation.sample(0, 0.2, false, 1))
-    }
-    val sparkPlan = sparkTestRelation.sample(0, 0.2, false, 1)
-    comparePlans(connectPlan.analyze, sparkPlan.analyze, false)
+    val connectPlan = connectTestRelation.sample(0, 0.2, false, 1)
+    val sparkPlan = sparkTestRelation.sample(false, 0.2 - 0, 1)
+    comparePlans(connectPlan, sparkPlan)
   }
 
   test("column alias") {
-    val connectPlan = {
-      import org.apache.spark.sql.connect.dsl.expressions._
-      import org.apache.spark.sql.connect.dsl.plans._
-      transform(connectTestRelation.select("id".protoAttr.as("id2")))
-    }
-    val sparkPlan = sparkTestRelation.select($"id".as("id2"))
-    comparePlans(connectPlan.analyze, sparkPlan.analyze, false)
+    val connectPlan = connectTestRelation.select("id".protoAttr.as("id2"))
+    val sparkPlan = sparkTestRelation.select(Column("id").alias("id2"))
+    comparePlans(connectPlan, sparkPlan)
   }
 
-  test("Aggregate with more than 1 grouping expressions") {
-    val connectPlan = {
-      import org.apache.spark.sql.connect.dsl.expressions._
-      import org.apache.spark.sql.connect.dsl.plans._
-      transform(connectTestRelation.groupBy("id".protoAttr, "name".protoAttr)())
-    }
-    val sparkPlan = sparkTestRelation.groupBy($"id", $"name")()
-    comparePlans(connectPlan.analyze, sparkPlan.analyze, false)
-  }
+// TODO: improve aggregate API parity.
+//  test("Aggregate with more than 1 grouping expressions") {
+//    val connectPlan =
+//      connectTestRelation.groupBy("id".protoAttr, "name".protoAttr)()
+//    val sparkPlan =
+//      sparkTestRelation.groupBy(Column("id"), Column("name")).agg(Map.empty[String, String])
+//    comparePlans(connectPlan, sparkPlan)
```

Comment:
Current Connect does not be compatible with existing DataFrame API on grouping/aggregation.